File: 2223/2223.cs
public class Solution {
    public int SumOfScoresOfBuiltStrings(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(SumOfScoresOfBuiltStrings(root.left), SumOfScoresOfBuiltStrings(root.right));
    }
}

File: 2223/2223.py
class Solution:
    def sum_of_scores_of_built_strings(self, root: Optional[TreeNode]) -> int:
        if not root:
            return 0
        return 1 + max(self.sum_of_scores_of_built_strings(root.left), self.sum_of_scores_of_built_strings(root.right))